### ISSUE TITLE
Can't verify image signature

### ISSUE THREAD

**Issue by LMantovan** [prometheus-operator/prometheus-operator#6130]
**What happened?**
I know that images are signed by cosign but I can't find how to verify it. Do i need a public key? Where can I find it?

**Did you expect to see something different?**
I expect a guidance page on documentation

**How to reproduce it (as minimally and precisely as possible)**:

**Environment**

* Prometheus Operator version:

    `Insert image tag or Git SHA here`
    
    

* Kubernetes version information:

    `kubectl version`
    

* Kubernetes cluster kind:

    insert how you created your cluster: kops, bootkube, etc.

* Manifests:

```
insert manifests relevant to the issue
```

* Prometheus Operator Logs:

```
insert Prometheus Operator logs relevant to the issue here
```

**Anything else we need to know?**:

**Comment by simonpasquier** [comment]
Yes the release pipeline uses keyless signing. You can verify the v0.69.1 image like this:

```
cosign verify quay.io/prometheus-operator/prometheus-operator:v0.69.1 \
  --certificate-identity=https://github.com/prometheus-operator/prometheus-operator/.github/workflows/publish.yaml@refs/tags/v0.69.1 \
  --certificate-oidc-issuer=https://token.actions.githubusercontent.com
````
